src/main/resources/data/tconstruct/scripts/tconstruct/casting_basin.zs

```diff
@@ -0,0 +1,35 @@
+/*
+ * Adds four Casting Basin recipes that do the following:
+ * 
+ * 1) Adds an Item Casting Recipe that, when a Honey Bottle is used as a Cast for 1000mb of Blood (Fluid), it gets cast into a Diamond of 200 ticks of cooling time. The Honey Bottle is not consumed and does not switch slots with the Diamond.
+ * 2) Adds a Material Casting Recipe that will allow a Diamond to be used as a Cast to make any Axe Head (Part), regardless of material (The Diamond acts as an Axe Head Cast). The Diamond is consumed by the recipe and does not switch slots with the Axe Head.
+ * 2.1) The result Item needs to be an Item that works with Materials, you can find a list of valid items by running `/ct dump ticMaterialItems`.
+ * 3) Adds a Composite Casting Recipe that uses Iron Tool Parts (Iron Axe Head for example) and Water (Fluid) to create a Bone version of that part. The recipe has a cooling temperature of 0°C. 
+ * 3.1) Composite Casting Recipes are Recipes where the Fluid is Cast onto a pre-existing tool part to create a new part.
+ * 3.2) You can find valid materialIds by running `/ct dump ticMaterials` in-game.
+ * 4) Adds a Container Filling Recipe that will fill a Copper Can with 100mb of any Fluid when cast.
+ * 4.1) This script is commented out as it conflicts with default Tinkers Recipes.
```

Comment:
Thats not going to work anyways. The number is just a hint to how much fluid we need to consume, it does not overcome the fact that containers need a specific number of mb input

Reply:
So you could add a recipe that would use 100mb but fill a bucket fully?

Reply:
No, you cannot. The recipe would fail to fill the container, but I think it would consume the fluid. The main reason for the number is so you can set a sane amount for variable size containers, like if you wanted to try to fill a thermal expansion portal tank.

For something in the mod to try, maybe a pickaxe? You can add a tank modifier to them, which accepts variable amounts of fluid.